### DISCUSSION TITLE
【求助】有段时间没登绳网了，为什么之前收藏的帖子都被删除了？

### DISCUSSION THREAD

**Discussion by Nie7bai** [share121/inter-knot#5146] [灌水]
有没有佬知道为什么？<img src="https://img-static.mihoyo.com/communityweb/upload/9249bd36125b953ddd445b19151427d6.png" alt="安比 疑惑" width="75" >

**Comment by 1luik** [comment]
可能是因为之前的绳网被黑

**Comment by Dave-12138** [comment]
#4569 <img src="https://bbs-static.miyoushe.com/static/2024/06/19/c638e43d0e2701e8f3e29a82dfd659fe_7613886348305980822.png" alt="流萤 没钱" width="75" >

**Reply by Nie7bai** [reply]
搜嘎<img src="https://bbs-static.miyoushe.com/static/2024/06/19/8701a3b5fc90a6c37ba5491913b8218f_2209157026344411896.png" alt="布洛妮娅 开心鸭" width="75" >
